components/commands/common/accelerator_pref_manager_unittest.cc

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commands/common/accelerator_pref_manager.h"
+#include "base/containers/contains.h"
```

Comment:
super nit:

empty line
```suggestion

#include "base/containers/contains.h"
```